tasks/eth/022-holocene-fp-upgrade/.env

```diff
@@ -8,3 +8,6 @@ SAFE_NONCE_0XC2819DC788505AAC350142A7A707BF9D03E3BD03=8  # noop
 SAFE_NONCE_0X847B5C174615B1B7FDF770882256E2D3E95B9D92=11 # +1, task 021
 
 SIMULATE_WITHOUT_LEDGER=0 # 1
```

Comment:
We need to remove this since the just file doesn't check if the value is `0` or `1` cf https://github.com/ethereum-optimism/superchain-ops/issues/448

Reply:
Removed and will make sure to not use it/remove it from other tasks as well. Note that I didn't add it in this PR but it got already merged with task 022.